<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidget</class>
 <widget class="QWidget" name="ListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_change">
       <property name="text">
        <string>change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove">
       <property name="text">
        <string>remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/delete.png</normaloff>:/resource/delete.png</iconset>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
   <property name="toolTip">
    <string>delete</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/new.png</normaloff>:/resource/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>new</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
